sample/Assets/Scripts/ZkEvmGetBalance.cs

```diff
@@ -0,0 +1,79 @@
+using System.Globalization;
+using System.Numerics;
+using System.Collections.Generic;
+using System;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+using Immutable.Passport;
+using Immutable.Passport.Model;
+
+public class ZkEvmGetBalance : MonoBehaviour
+{
+#pragma warning disable CS8618
+    [SerializeField] private Text Output;
+    [SerializeField] private InputField AddressInput;
+
+    private Passport Passport;
+#pragma warning restore CS8618
+
+    void Start()
+    {
+        if (Passport.Instance != null)
+        {
+            // Get Passport instance
+            Passport = Passport.Instance;
+        }
+        else
+        {
+            ShowOutput("Passport instance is null");
+        }
+    }
+
+    /// <summary>
+    /// Gets the balance of the account for the specified address.
+    /// The balance is obtained in hexadecimal format and then converted to decimal for display.
+    /// </summary>
+    public async void GetBalance()

```

Comment:
This only gets the balance of the native token, right?

Reply:
Correct